### PR TITLE
Researcher page updates

### PR DESCRIPTION
**Pull request by abeconnelly**
- Added Q/A about where to find BAM/SAM/FASTQ/etc. files in researcher.md.
- Added some more pointers to Tapestry to search for data directly there instead of first pointing them to untap in researcher.md
- Added pointer to contact form in researcher.md for further questions (points to the PersonalGenomes.org contact form).
- Minor formatting and spelling changes all around (including the participant.md file).

### CODE REVIEW

File: researcher.md [hunk 1]
```diff
@@ -4,9 +4,25 @@
...
 
 # Researcher FAQ
-### Researcher Requests for PGP Data
 
-There is a consolidated database of the Harvard PGP available for download:
+### Data Access for Harvard PGP Data
+
+The Harvard Personal Genome Project participant page, located at [my.pgp-hms.org](https://my.pgp-hms.org), holds information on participant data.
+Each Harvard Personal Genome Project participant has a profile page listing their available information.
+For example, the [profile participant for hu43860C](https://my.pgp-hms.org/profile/hu43860C) has medical information,
```
Comment (madprime) [inline researcher.md:12]: Maybe also note that there's JSON-format data for profiles that facilitates automated retrieval of data from profile pages. For example: https://my.pgp-hms.org/profile/hu43860C.json

File: researcher.md [hunk 2]
```diff
@@ -16,4 +32,23 @@ The spreadsheets and SQLite3 database only represent most of the data provided b
 
 Source code for download and presentation of the above database can be found on the [untap](https://github.com/abeconnelly/untap) repository.
 
-Please feel free to email the Harvard Personal Genome Project staff if you have any questions or run into any problems.
+### Where can I get the FASTQ/BAM/VCF/etc. files?
+
+Most genomic data available through the Harvard Personal Genome Project site is only available in the format provided to us by Complete Genomics (CGI) and would need to be converted.
+
+CGI gives us unaligned read information, variant evidence files and CGI-Var files, among others.
```
Comment (madprime) [inline researcher.md]: I believe that, for a very large fraction of our genomes, do not have "unaligned read information". As I understand it, we elected to not receive the "MAP" directories in data sets in exchange for a lower price. Description of MAP directory here - http://www.completegenomics.com/documents/DataFileFormats_Standard_Pipeline_2.5.pdf

**Comment by abeconnelly** [comment]
- Added a Q/A about the `.json` extension (also mentioned the GET-Evidence report)
- Took out line about receiving unaligned reads from CGI
- Added in CC0 license (redundant but good to have here as well)

**Comment by celiafw** [comment]
Thank you Abram, that is just fantastic and will be super helpful going
forward.

Celia

**Comment by nouyang-curoverse** [comment]
I believe have merged these changes into the wiki, notably I changed the support email to the PGP Harvard form instead of personalgenomes.org contact form (seems more appropriate)? Thanks all!

**Pull request closed without merging**